mortgage/transform-data.py

```diff
@@ -15,7 +15,7 @@ def main():
     try:
         source, dest = [os.path.abspath(x) for x in sys.argv[1:]]
     except TypeError:
-        sys.exit("Usage: %s source-path dest-path" % sys.argv[0])
+        raise TypeError("Usage: %s source-path dest-path" % sys.argv[0])
```

Comment:
```suggestion
        raise ValueError("Usage: %s source-path dest-path" % sys.argv[0])
```

`TypeError` is mostly used for incorrect _type_ of stuff, not its value.
Also it's a bit unclear why `TypeError` is caught here, what is throwing it? I would have expected `ValueError` from tuple unpacking at line 16